src/features/utils/ansibleCfg.ts

```diff
@@ -29,13 +29,13 @@ export async function scanAnsibleCfg(
   rootPath: string | undefined = undefined
 ): Promise<AnsibleVaultConfig | undefined> {
   /*
-   * Reading order:
+   * Reading order (based on the documentation: https://docs.ansible.com/ansible/2.4/intro_configuration.html):
```

Comment:
This point to the latest stable release docs https://docs.ansible.com/ansible/latest/installation_guide/intro_configuration.html

Reply:
Also `ansible --version` has information about the ansible configuration file path read from.

Reply:
Thanks for pointing that out.
I can link this page then, that points to the locations: https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-configuration-settings

Reply:
Yes, I am aware of the `ansible --version` command, but this code is present on the extension side, and the dev who wrote this intends to scan the locations rather than using the command. Thus my PR is only a rectification for it.

Reply:
Yeah, when user does not have ansible we cannot call ansible itself to report config location.